.github/CODEOWNERS

```diff
@@ -32,6 +32,17 @@
 /sdcm/cloud_api_client.py   @dimakr
 /sdcm/cloud_api_utils.py    @dimakr
 
+# k8s
+/sdcm/cluster_k8s/          @abramche
+/sdcm/k8s_configs/          @abramche
+/defaults/k8s*.py           @abramche
+
+# Operator functionality
+/jenkins-pipelines/operator/                @abramche
+/test-cases/scylla-operator/                @abramche
+sdcm/fill_db_data.py                        @abramche
+upgrade_test.py                             @abramche
```

Comment:
are these two really run only on k8s? based on their name I would expect them to be used for normal scylla as well

Reply:
not only, but they are not in the k8s section. I can move these two elsewhere (but where? we do not seem to have a test structure for these)

`fill_db_data.py` is only used by `upgrade_test.py`

Reply:
it's both, add my name there as well, as the owner of scylla upgrades

if you think a refactor is needed, got right ahead, please don't  refactor only cause of aligning the code ownership.